loops.js

```diff
@@ -51,3 +66,14 @@
 // >> 9
 // ==========================================
 
+for (let i = 0; i < 10; i++) {
```

Comment:
for (let i = 0; i < 10; i++) {
  if (i < 3) {
    console.log(i);
  }
  if (i >= 3 && i < 6) {
    console.log('> ' + i);
  }
  if (i >= 6) {
    console.log('>> ' + i);
  }
}

// of, ook correct:
for (let i = 0; i < 10; i++) {
  if (i >= 3 && i < 6) {
    console.log('> ' + i);
  } else if(i >= 6) {
    console.log('>> ' + i);
  } else {
    console.log(i);
  }
}